src/status_im/ui/screens/wallet/components/views.cljs

```diff
@@ -162,10 +163,12 @@
 (defn- recipient-address [address modal?]
   [react/text {:style               (merge styles/recipient-address (when-not address styles/recipient-no-address))
                :accessibility-label :recipient-address-text}
-   (or (eip55/address->checksum (ethereum/normalized-address address))
-       (if modal?
-         (i18n/label :t/new-contract)
-         (i18n/label :t/specify-recipient)))])
+   (if (ens/is-valid-eth-name? address)
```

Comment:
seems like that this fixes #9357

Reply:
The overall PR does now fix #9357 as well though I'm not 100% sure if this particular line of code is useful or not.  I had originally added this line to make sure it didn't try to run an ENS name through the EIP55 checksum process since that's what generated the error @churik identified (and I saw as well).  Not sure if it's still useful or not since we're taking care of name resolution in the `fill-request-from-url` method that populates the send transaction now.